web-storage/README.md

```diff
@@ -0,0 +1,47 @@
+# Text
+
+Web Storage provides mechanisms to store key value pairs in a much more intuitive way than using cookies.
+
+There are two different APIs available:
+
+- sessionStorage
+
+  - The values are stored only for the duration of page session. The values get cleared when the tab is closed.
```

Comment:
"The values are stored only for the duration of page session"
- Can be explained in more detail in one or two sentences. Like: what is page session?

Reply:
@avishekjana I have updated the content